200.html

```diff
@@ -1,9 +1,148 @@
-<!doctype html>
+<!DOCTYPE html>
 <html data-n-head="">
   <head data-n-head="">
-    <title data-n-head="true">OwlHacks - Temple University&#x27;s Premier Hackathon</title><meta data-n-head="true" charset="utf-8"/><meta data-n-head="true" name="viewport" content="width=device-width, initial-scale=1"/><meta data-n-head="true" data-hid="description" name="description" content="Temple University&amp;#39;s Hackathon Website"/><link data-n-head="true" rel="icon" type="image/x-icon" href="/owlhacks-icon.png"/><link rel="preload" href="/_nuxt/a9319e3cd0013af34655.js" as="script"><link rel="preload" href="/_nuxt/d368e7381e4970061e75.js" as="script"><link rel="preload" href="/_nuxt/93ab85c19c5f7acf8687.js" as="script"><link rel="preload" href="/_nuxt/54c6922fb61b4818893d.js" as="script">
+    <title data-n-head="true">
+      OwlHacks - Temple University&#x27;s Premier Hackathon
+    </title>
+    <meta data-n-head="true" charset="utf-8" />
+    <meta
+      data-n-head="true"
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    />
+    <meta
+      data-n-head="true"
+      data-hid="description"
+      name="description"
+      content="Temple University&amp;#39;s Hackathon Website"
+    />
+    <link
+      data-n-head="true"
+      rel="icon"
+      type="image/x-icon"
+      href="/owlhacks-icon.png"
+    />
+    <link rel="preload" href="/_nuxt/a9319e3cd0013af34655.js" as="script" />
+    <link rel="preload" href="/_nuxt/d368e7381e4970061e75.js" as="script" />
+    <link rel="preload" href="/_nuxt/93ab85c19c5f7acf8687.js" as="script" />
+    <link rel="preload" href="/_nuxt/54c6922fb61b4818893d.js" as="script" />
```

Comment:
i don't think this site uses nuxt stuff anymore. i guess it's carryover from when it was converted from before. probably worth getting rid of it.

Reply:
> are you taking over OwlHacks? i can probably add you as an admin to the repo, though I am not an organization admin.

yeah I'm taking it over or 2023! Have to make some changes so the website is up to date to share with sponsors asap! 👍 

> i don't think this site uses nuxt stuff anymore. i guess it's carryover from when it was converted from before. probably worth getting rid of it.

I'm just getting familiar with the website now and I haven't touched much besides some text in the html files and commenting out/in some stuff. We aren't MLH certified this year so had to take out those banners. Besides that I just used a formatter from vscode on a few files.

Reply:
Sounds good! You can do whatever you want to the site - redesign it, go with whats there, whatever - have fun!